### ISSUE TITLE
Permission not granted if no child is selected (UI)

### ISSUE THREAD

**Issue by nev-21** [abpframework/abp#14551]
### Documentation

https://docs.abp.io/en/abp/latest/Authorization#child-permissions

```
var authorManagement = myGroup.AddPermission("Author_Management");
authorManagement.AddChild("Author_Management_Create_Books");
authorManagement.AddChild("Author_Management_Edit_Books");
authorManagement.AddChild("Author_Management_Delete_Books");
```
> -GetListAsync and GetAsync will be available to users if they have Author_Management permission is granted.
> -Other methods require additional permissions.

### GitHub Issues

* **ABP Framework 6.0.0**.
* Angular 14.2.8
* EF Core

### Steps to reproduce

1. Create a simple app with a permission with childs (You can also use the abp online demo).
2. Check only the parent permission.
3. See the number of granted permissions is not including the permission.
4. Save and open again
5. The permission was not granted (saved)

Example: see Languages permission with and without child, after click Save it is not saved.

![image](https://user-images.githubusercontent.com/89469783/200143855-fe9c0b66-c131-4e6b-adbf-d63c3ddaa8d3.png)

It doesn't recognize the permission, the number should be (3).

![image](https://user-images.githubusercontent.com/89469783/200143868-c5f4aafe-f5cb-43d5-8bef-e44b3b42c289.png)

After click on save, it is not saved.
![image](https://user-images.githubusercontent.com/89469783/200143900-63b40297-6034-4172-ad8c-0b36de8708ec.png)

**Comment by phil-enfom** [comment]
Hi, 
Also have this issue on ABP 7.0 with Angular UI
If we want to just give just readonly access to a user we would only tick the parent permission but on save this is lost.

Can I have an update on this please.

Phil

**Comment by phil-enfom** [comment]
@maliming this has been resolved by #14942
